app/src/main/res/values/arrays.xml

```diff
@@ -1,15 +1,147 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string-array name="countryArray">
```

Comment:
Is there specific reason for having two arrays as country names and country array? Doesn't both these arrays reflects the same content??